docs/guides/parallel-scraping/parallel-scraping.mdx

```diff
@@ -0,0 +1,220 @@
+and will fork copies of itself to do the scraping.
+
+#### Why do we create a Promise per worker process?
+
+We use this to ensure the parent process stays alive until all the worker processes exit. Otherwise, the worker processes would just get spawned, and lose the ability
+to communicate with the parent. You might not need this depending on your use case (maybe you just need to spawn workers and let them process).
+
+#### What's with all those `Configuration` calls?
+
+There are three steps we want to do for the worker processes:
+
+- ensure the default storages do **not** get purged on start, as otherwise we'd lose the queue we prepared
+- get the queue that supports locking from the same location as the parent process
+- initialize a special storage for worker processes so they do not collide with each other
+
+In order, that's what these lines do:
+
+```javascript title="src/parallel-scraper.mjs"
+// Disable the automatic purge on start (step 1)
+// This is needed when running locally, as otherwise multiple processes will try to clear the default storage (and that will cause clashes)
+Configuration.getGlobalConfig().set('purgeOnStart', false);
+
+// Get the request queue from the parent process (step 2)
+const requestQueue = await getOrInitQueue(false);
+
+// Configure crawlee to store the worker-specific data in a separate directory (needs to be done AFTER the queue is initialized when running locally) (step 3)
+const config = new Configuration({
+    storageClientOptions: {
+        localDataDirectory: `./storage/worker-${process.env.WORKER_INDEX}`,
+    },
+});
+```
+
+#### Enabling the request locking experiment, and telling the crawler to use the worker configuration
+
+You might have noticed several lines highlighted in the code above. Those show how you can enable the request locking experiment, as well as how you provide the request queue to the crawler.
+You can read more about the experiment by visiting the [request locking experiment](../experiments/experiments-request-locking) page.
+
+You might have also noticed we passed in a second parameter to the constructor of the crawler, the `config` variable we created earlier.
+This is needed to ensure the crawler uses the worker-specific storages for internal states, and that they do not collide with each other.
+
+#### Why do we use `process.send` instead of `context.pushData`?
+
+Since we use child processes, and each worker process has its own storage space, calling `context.pushData` will not work as we want it to work (each worker would just
+push to its own personal dataset that is considered the "default" one). Instead, we need to send the data back to the parent process, which has the dataset where we want to
+store the data, in a centralized place.
+
+:::tip Why don't we apply the same logic we did to the request queue to the dataset?
```

Comment:
is tip the best type of admonition to use here? note or info seem more appropriate

Reply:
I'll go with info